druid/src/widget/textbox.rs

```diff
@@ -250,7 +269,12 @@ impl<T: TextStorage + EditableText> Widget<T> for TextBox<T> {
                 self.editor.set_text(data.to_owned());
                 self.editor.rebuild_if_needed(ctx.text(), env);
             }
-            LifeCycle::FocusChanged(_) => {
+            LifeCycle::FocusChanged(_is_focused) => {
+                #[cfg(target_os = "macos")]
+                if *_is_focused && !self.was_focused_from_click {
+                    self.editor.select_all(data);
+                }
```

Comment:
Why not use `cfg!` and avoid the `_`?
```suggestion
            LifeCycle::FocusChanged(is_focused) => {
                if cfg!(target_os = "macos") && *is_focused && !self.was_focused_from_click {
                    self.editor.select_all(data);
                }
```